<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="use_sim_time" default="true"/>
  <arg name="vehicle_model" default="sample_vehicle"/>
  <arg name="rviz_config_name" default="autoware.rviz"/>
  <arg name="rviz_config_path" default="$(find-pkg-share autoware_launch)/rviz/$(var rviz_config_name)"/>

  <!-- Global parameters -->
  <group scoped="false">
    <include file="$(find-pkg-share autoware_global_parameter_loader)/launch/global_params.launch.py">
      <arg name="use_sim_time" value="$(var use_sim_time)"/>
      <arg name="vehicle_model" value="$(var vehicle_model)"/>
    </include>
  </group>

  <!-- RViz -->
  <node pkg="rviz2" exec="rviz2" name="rviz2" args="-d $(var rviz_config_path)"/>
</launch>